Packs/CloudIncidentResponse/Playbooks/playbook-Cloud_Token_Theft_-_Set_Verdict.yml

```diff
@@ -250,9 +250,7 @@ tasks:
       - - operator: containsGeneral
           left:
             value:
-              complex:
-                root: Core.OriginalAlert.[0].event
-                accessor: caller_ip_asn_org
+              simple: Core.OriginalAlert.event.caller_ip_asn_org
```

Comment:
Will it work? 
Because its "simple" - please verify that it takes the context path value and not the input (the context path) as the value

Reply:
Yes. Looks good.